23_multi-assay_analyses.Rmd

```diff
@@ -71,17 +71,17 @@ mae[[1]] <- mae[[1]][!is.na(rowData(mae[[1]])$Phylum), ]
 rowData(mae[[1]]) <- DataFrame(apply(rowData(mae[[1]]), 2, 
                                      str_remove, pattern = "._[0-9]__"))
 # Microbiome data
-mae[[1]]
+getWithColData(mae, 1)
```

Comment:
Instead of indices, use the experiment names "microbiota", "metabolites", "biomarkers".. for clarity. 

Before that you can show how to check what experiments there are, with `experiments(mae)`